backend/src/files/files.controller.ts

```diff
@@ -42,4 +52,27 @@ export class FilesController {
 			statusCode: 302,
 		};
 	}
+
+	@Post("export-markdown")
+	@ApiOperation({
+		summary: "Export Markdown",
+		description: "Export Markdown to various formats",
+	})
+	@ApiBody({ type: ExportFileRequestBody })
+	@ApiResponse({ status: 200, description: "File exported successfully" })
+	async exportMarkdown(
+		@Body() exportFileRequestBody: ExportFileRequestBody
+	): Promise<StreamableFile> {
+		try {
+			const { fileContent, mimeType, fileName } =
+				await this.filesService.exportMarkdown(exportFileRequestBody);
+
+			return new StreamableFile(fileContent, {
+				type: mimeType,
+				disposition: `attachment; filename="${fileName}"`,
+			});
+		} catch (error) {
+			throw error;
+		}
```

Comment:
It seems that the try-catch can be removed.
This code achieves the same effect as delegating error handling to NestJS.

Reply:
Fix associate issues with [this commit](https://github.com/yorkie-team/codepair/pull/238/commits/4cedb0459d310378ed05fd9554fa352eb16316ef).